static/js/components/LoginPopup.js

```diff
@@ -45,9 +44,4 @@ export class LoginPopupHelper extends React.Component<LoginPopupProps> {
   }
 }
 
-export default R.compose(
-  connect(
-    R.always({}) // mapStateToProps is not needed - just return an object
-  ),
-  onClickOutside
-)(LoginPopupHelper)
+export default R.compose(onClickOutside)(LoginPopupHelper)
```

Comment:
I think this can now just be `onClickOutside(LoginPopupHelper)`, unless I'm mistaken